CommunityShedMVC/src/CommunityShedMVC/Views/Account/Register.cshtml

```diff
@@ -0,0 +1,34 @@
+﻿@model CommunityShedMVC.ViewModels.RegisterViewModel
+
+@{
+    ViewBag.Title = "Register";
+}
+
+<h2>Register</h2>
+
+@using (Html.BeginForm())
+{
+    @Html.AntiForgeryToken()
```

Comment:
Again, don't forget to add a `ValidateAntiForgeryToken` attribute to the POST action method.